tests/src/test/scala/whisk/core/cli/test/WskWebActionsTests.scala

```diff
@@ -114,4 +114,56 @@ class WskWebActionsTests
             authorizedResponse.statusCode shouldBe 200
             authorizedResponse.body().asString() shouldBe namespace
     }
+
+    it should "fail to invoke web action with an unsupported content type when x-ow-raw-http header is not supplied" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "webaction"
+            val file = Some(TestUtils.getTestActionFilename("echo.js"))
+            val bodyContent = "This is the body"
+
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) =>
+                    action.create(name, file, annotations = Map("web-export" -> true.toJson))
+            }
+
+            val host = getServiceURL()
+            val url = host + s"/api/v1/experimental/web/$namespace/default/webaction.text"
+            val response = RestAssured.given().header("x-ow-raw-http", "false").contentType("text/html").body(bodyContent).config(sslconfig).post(url)
+            response.statusCode shouldBe 401
```

Comment:
The 401 status code is a defect fixed here: https://github.com/openwhisk/openwhisk/pull/1939. Test will have to be updated once that PR is merged.